apps/frontend/assets/src/omg-create-blockchain-transaction-modal/index.js

```diff
@@ -99,9 +99,14 @@ class CreateBlockchainTransaction extends Component {
 
   setGasPrice = async () => {
     const { estimateGasFromTransaction } = this.props
+    if (this.state.toPlasma) {
+      return this.setState({ gasPrice: 500 })
```

Comment:
Should we move the price out in to a common class?

Also why 500?

Reply:
@unnawut unlike the eth transfer I don't know how to estimate the cost of transfer for a contract call so the 500 is a pretty random default. @jarindr maybe has an idea to estimate contract call gas? I don't know what amount would be a better default?